### PR TITLE
feat(info): add optional custom format

### PR DESCRIPTION
**Pull request by adamreese**
Allow for custom formatting of `helm info` using a Go template

Replaces: https://github.com/deis/helm/pull/58

### CODE REVIEW

**Comment by technosophos** [comment]
The tests aren't passing

**Comment by carmstrong** [comment]
This also needs a rebase now, @adamreese

**Comment by adamreese** [comment]
@technosophos it's an issue of other unit tests changing the current working directory.  I have a fix in progress

File: helm/action/helper_test.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	// create cache directory
+	tmpHomeCache := filepath.Join(tmpHome, "cache")
+	os.Mkdir(tmpHomeCache, dstat.Mode())
+
+	// absolute path to testdata charts
+	testChartsPath := filepath.Join(helmRoot, "helm/testdata/charts")
+
+	// copy testdata charts into cache
+	// mock git clone
+	exec.Command("cp", "-R", testChartsPath, tmpHomeCache).Run()
```
Comment (lanej) [inline helm/action/helper_test.go]: no golang lib function for this ?

Comment (adamreese) [inline helm/action/helper_test.go]: noop, you need to walk the dir and copy each file

Comment (technosophos) [inline helm/action/helper_test.go]: There's a copyDir() function in (I think) `action/update.go` that does essentially this.

**Comment by technosophos** [comment]
I'd prefer using the `copyDir()` func if it suits your needs, but I'm going to go ahead and LGTM this.

File: helm/action/helper_test.go (hunk 1 above)
Comment (adamreese) [inline helm/action/helper_test.go]: oh I missed that, thanks

**Pull request merged**